include/wx/aui/auibar.h

```diff
@@ -125,7 +141,9 @@ class WXDLLIMPEXP_AUI wxAuiToolBarItem
         m_alignment = wxALIGN_CENTER;
     }
 
-    void Assign(const wxAuiToolBarItem& c)
+    virtual ~wxAuiToolBarItem() {}
```

Comment:
A nitpick but this should be
```suggestion
    virtual ~wxAuiToolBarItem() = default;
```